packages/golang1/packaging

```diff
@@ -2,5 +2,5 @@
 
 set -e
 
-tar xzf golang/go1.10.linux-amd64.tar.gz
+tar xzf golang/go1.13.7.linux-amd64.tar.gz
```

Comment:
We can (optionally) use a wildcard on the version; as long as the `spec` file has the file name, it should work fine.